tests/model_explainability/evalhub/kueue/conftest.py

```
@@ -0,0 +1,473 @@
+
+
+def _is_evalhub_crd_available(admin_client: DynamicClient) -> bool:
+    """Check if EvalHub CRD is installed on the cluster."""
+    crd_name = "evalhubs.trustyai.opendatahub.io"
+    try:
+        crd = CustomResourceDefinition(
+            client=admin_client,
+            name=crd_name,
+        )
+        return crd.exists
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)